gitlab-runner/win-values.yaml

```diff
@@ -2,22 +2,31 @@ rbac:
  create: true
  rules:
  - apiGroups: [""]
-   resources: ["pods", "secrets", "configmaps"]
-   verbs: ["get", "list", "watch", "create", "patch", "delete", "update"]
- - apiGroups: [""]
-   resources: ["pods/exec", "pods/attach"]
+   resources: ["pods/attach", pods/exec]
    verbs: ["create", "patch", "delete"]
  - apiGroups: [""]
-   resources: ["pods/log"]
+   resources: ["pods"]
+   verbs: ["get", "list","watch", "create", "patch", "delete"]
+ - apiGroups: [""]
+   resources: ["services"]
+   verbs: ["get", "list", "watch", "create", "patch", "delete"]
+ - apiGroups: [""]
+   resources: ["configmaps"]
+   verbs: ["get", "create", "update", "delete"]
+ - apiGroups: [""]
+   resources: ["secrets"]
+   verbs: ["get", "list", "watch", "create", "update","patch", "delete"]
+ - apiGroups: [""]
+   resources: ["serviceAccounts"]
```

Comment:
For me it seems much over permissive, please explain why they need all this permissions.
I would like to see usage of API Groups.

Reply:
added to Monday, holding PR.

Reply:
﻿﻿https://github.com/SygniaLabs/ScallOps/pull/22/commits/843b124ece169f2db422b388950b32b1aaba1b09